fullstack-cert/html-projects/lab-multimedia-player/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Multimedia Player</title>
+</head>
+<body>
+    <h1>Multimedia Player</h1>
+    <section>
+        <h2>Now Playing: Symphony No. 5</h2>
+        <audio controls aria-label="Audio Player for Symphony No. 5">
```

Comment:
do we need the aria-label here?
I am actually not sure. same question for the video element later on.

Could the aria-label be removed for both of these elements @bbsmooth ?